scripts/variantstore/wdl/GvsFindHailRefuse.wdl

```diff
@@ -0,0 +1,132 @@
+version 1.0
+
+import "GvsUtils.wdl" as Utils
+
+workflow GvsFindHailRefuse {
+    input {
+        String? submission_id_to_search
+        Boolean perform_deletion = false
+    }
+    parameter_meta {
+        submission_id_to_search: "If defined, the submission id in the current workspace under which to search for Avro files, useful for testing against specific integration test runs. If undefined, search all submissions in the current workspace."
```

Comment:
does this mean it might clean up everything (including wanted stuff) if someone forgets to fill this out?

Reply:
If `perform_deletion` were overridden to `true` that's what would happen, which is why I defaulted `perform_deletion` to `false`. tbh I would not mind removing the actual deletion code from this WDL and having it just report whatever cruft it finds, if any.